File: swagger/models/get_client_referral_types_response.py
from .base_model import BaseModel


class GetClientReferralTypesResponse(BaseModel):
	"""
	no description available

	Attributes:

		referral_types
		The list of available referral types.
	"""

	input_map = {
		'ReferralTypes': 'referral_types',
		}

	output_map = {
		'referral_types': 'ReferralTypes',
		}

	referral_types = None

File: swagger/Models/GetClientReferralTypesResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetClientReferralTypesResponse extends BaseModel
{
	/**
	 * The list of available referral types.
	 * 
	 * @var string[] 
	 */
	public $referralTypes;

	protected function getInputMap()
	{
		return [
			'ReferralTypes' => ['referralTypes', 'array', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'referralTypes' => ['ReferralTypes', 'array', null],
			];
	}
}
